components/LinkSummary.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC } from 'react'
+import Link from 'next/link'
+
+export interface LinkSummaryItem {
+  href: string
+  text: string
+}
+
+export interface LinkSummaryProps {
+  id: string
+  links: LinkSummaryItem[]
+}
+
+const LinkSummary: FC<LinkSummaryProps> = ({ id, links }) => {
+  return (
+    <section id={id}>
+      <ul className="list-none pb-6 ml-4">
```

Comment:
I think this should be styled like a normal list with bullets; and would be good to include a `p` tag with some text above, for now can give it the text "Other way to get your status:" (translated)